packages/transaction-controller/src/types.ts

```diff
@@ -1307,6 +1307,9 @@ export type SimulationError = {
 
 /** Simulation data for a transaction. */
 export type SimulationData = {
+  /** Set to true if simulationData changed after updating security alert */
+  changeInSimulationData?: boolean;
```

Comment:
What is the purpose of this property?

So the client can trigger an additional alert if the simulation isn't consistent?

If so, how do we differentiate between expected updates if the user changes the `value` or `data`?

Is this needed at all?

Reply:
[According to the designs](https://www.figma.com/design/aHdSNvg9ahXZMtThzz3bUo/Github-issues?node-id=650-23456&node-type=frame&t=gQRhvfuWqwn5OdbV-0), UI need to know if that re-simulation done because of the security alert. 
I didn't have a chance to test this one out yet, so please consider this as draft for now.